### ISSUE TITLE
CSS gap, column-gap, row-gap - missing top-level `_compat`

### ISSUE THREAD

**Issue by meyerweb** [mdn/browser-compat-data#11559]
The following files are missing a top-level `_compat` object:

* https://github.com/mdn/browser-compat-data/blob/main/css/properties/gap.json
* https://github.com/mdn/browser-compat-data/blob/main/css/properties/column-gap.json
* https://github.com/mdn/browser-compat-data/blob/main/css/properties/row-gap.json

This appears to be breaking the “Specifications” macros on the following MDN pages:

* https://developer.mozilla.org/en-US/docs/Web/CSS/gap#specifications
* https://developer.mozilla.org/en-US/docs/Web/CSS/column-gap#specifications
* https://developer.mozilla.org/en-US/docs/Web/CSS/row-gap#specifications

I considered just adding top-level `_compat` with a `spec_url` and leaving it at that, but can’t test MDN macro resolution on my local install at present, so can’t test whether that breaks other things, so decided to file an issue here instead.

If someone can point me at what exactly I need to add to make this work without breaking other stuff, I can make the edits and submit a PR myself.

**Comment by meyerweb** [comment]
Following up: I got a local copy of everything up and running, including working off my local branch of BCD.  After a fair bit of experimentation, I discovered that adding a top-level `__compat` to the beginning of `gap.json` as follows…

```
{
  "css": {
    "properties": {
      "gap": {
        "__compat": {
          "spec_url": "https://drafts.csswg.org/css-align/#gap-shorthand",
          "support": {
          }
        },
```

…was the minimum I could add in and get the specification link to appear while not breaking either the BCD table, or the page as a whole.  This did cause the BCD table to add a row labeled `gap` with all question marks, but the rest of the BCD table was correct.  If I remove the `support` object, the page fails to render, as it would seem the document building process requires its presence in a `__compat` object.

Is there another way to add specification URLs to BCD objects, one that sidesteps the support requirement?  If not, I can add support information to the top-level object at the cost of prepending a row to the BCD table, but that strikes me as inelegant and fragile over long time scales.
